cypress/support/commands.js

```diff
@@ -8,34 +8,42 @@
 // https://on.cypress.io/custom-commands
 // ***********************************************
 
-import '@testing-library/cypress/add-commands';
-import { configure } from '@testing-library/cypress';
 import { register } from '@cypress/snapshot';
+import { configure } from '@testing-library/cypress';
+import '@testing-library/cypress/add-commands';
 
 register();
 configure({ testIdAttribute: 'data-test-id' });
 
+const getIframeWindow = () => {
```

Comment:
that's the same as in cypress/fixtures/utils, can't we reuse it?

Reply:
Done